### PR TITLE
Demo 2499 tutorial single host

### PR DESCRIPTION
**Pull request by gabeobrien**
Work in progress on adding Tutorials!

### CODE REVIEW

File: tutorials/tutorial_install_a_service/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+# Demo Catalog : Tutorial : Install a Service
+
+If you have not got the demo-deployer setup please follow our [getting started guide](/GETTING_STARTED.md) and get your AWS credential setup before working through this tutorial.
+
+# Tutorial : How to build a demo: Installing a Service
+
+In this second tutorial we will be building on the [first tutorial](../tutorial_provision_a_host) provision a host and then installing a service on the host.  Anything that will be installed on a host is defined in block called "services" within a deployment configuration file.
+
+[Example deploy configuration](single-sevice.json):
```
Comment (nr-kkenney) [inline tutorials/tutorial_install_a_service/README.md:11]: typo for 'service'

File: tutorials/tutorial_install_a_service/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+| destinations       | the hosts that this service will be installed on |
+
+
+## Installing the single service on a host
+
+```
+docker run -it\
+    -v $HOME/demo-deployer/configs/:/mnt/deployer/configs/\
+    --entrypoint ruby ghcr.io/newrelic/deployer main.rb\
+    -d https://raw.githubusercontent.com/newrelic/demo-catalog/DEMO-2499-tutorial-single-host/tutorials/tutorial_single_host/single-service.json
```
Comment (nr-kkenney) [inline tutorials/tutorial_install_a_service/README.md:56]: the folder 'tutorial_single_host' doesnt exist

File: tutorials/tutorial_install_a_service/single-sevice.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (nr-kkenney) [inline tutorials/tutorial_install_a_service/single-sevice.json:1]: type in filename: 'single-sevice.json'

File: tutorials/tutorial_install_a_service/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+In the above configuration we define a single service that will be installed on the 'compute' host.  Here is a quick summary of what each of the service fields mean:
+
+| Field Name         |  Field Description |
+| ------------------ | ------------------ |
+| id                 | is used for referring to the service for defining relationships and for instrumentation targeting |
+| display_name       | is used by instrumentation as the services name  |
+| source_repository  | the local path or git repo where the code for the service lives |
+| deploy_script_path | the location of the ansibe play that will be invoked to actually install the service |
```
Comment (nr-kkenney) [inline tutorials/tutorial_install_a_service/README.md:44]: typo in 'ansibe'

File: tutorials/tutorial_install_a_service/README.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  ]
+}
+```
+
+In the above configuration we define a single service that will be installed on the 'compute' host.  Here is a quick summary of what each of the service fields mean:
+
+| Field Name         |  Field Description |
+| ------------------ | ------------------ |
+| id                 | is used for referring to the service for defining relationships and for instrumentation targeting |
+| display_name       | is used by instrumentation as the services name  |
```
Comment (nr-kkenney) [inline tutorials/tutorial_install_a_service/README.md:42]: i think you mean "as the service name" here.

File: tutorials/3_instrument_resource_and_service/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+[![Experimental Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#experimental)
+
+# Demo Catalog : Tutorial : Instrument a Resource and Service
+
+If you have not got the demo-deployer setup please follow our [getting started guide](/GETTING_STARTED.md) and get your AWS and New Relic credential setup before working through this tutorial.
+
+# Tutorial : How to build a demo: Instrumentation a resource and service
+
+In this third tutorial we will be creating a deploy configuration that will provision a single resource, install a service on that resource and setup New Relic instrumentation on both.  All instrumentation is defined in a block called "instrument" within a deployment configuration JSON file.  The demo-deployer will install and setup the New Relic instrumentation using the user configuration credentials you provided in the demo-deployer setup.
```
Comment (aswanson-nr) [inline tutorials/3_instrument_resource_and_service/README.md:9]: `All instrumentation is defined in a block called "instrument" within a deployment configuration JSON file.` should be update to `All instrumentation is defined in a block call "instrumentation" within a deployment configuration JSON file.`

File: tutorials/3_instrument_resource_and_service/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+      "id": "nodetron",
+      "display_name": "Nodetron Service",
+      "source_repository": "https://github.com/newrelic/demo-nodetron.git",
+      "deploy_script_path": "/deploy/linux/roles",
+      "port": 5000,
+      "relationships": [],
+      "destinations": ["compute"]
+    }
+  ],
+	"instrumentations": {
```
Comment (aswanson-nr) [inline tutorials/3_instrument_resource_and_service/README.md:35]: Some weird formatting here.

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**